specutils/spectra/spectrum_list.py

```diff
@@ -0,0 +1,13 @@
+from astropy.nddata import NDIOMixin
+
+
+__all__ = ['SpectrumList']
+
+
+class SpectrumList(list, NDIOMixin):
+    """
+    A list that is used to hold a list of Spectrum1D objects
+
+    The primary purpose of this class is to allow loaders to return a list of
+    heterogenous spectra that do have a spectral axis of the same length.
```

Comment:
As above, might be worth specifically saying that it is a list with the `NDIOMixin` added in.  It's obvious when I look at the code, but it's not as clear when one sees the docs (or if one were looking at the docstring  via `help(SpectrumList)` or similar.

Reply:
I think I accomplished this in a more generic way without referring to implementation details.